<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NERoomDataWidget</class>
 <widget class="QWidget" name="NERoomDataWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>408</width>
    <height>224</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>408</width>
    <height>224</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>408</width>
    <height>224</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>24</number>
   </property>
   <property name="leftMargin">
    <number>24</number>
   </property>
   <property name="topMargin">
    <number>24</number>
   </property>
   <property name="rightMargin">
    <number>24</number>
   </property>
   <property name="bottomMargin">
    <number>24</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>8</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="lblDelay">
        <property name="text">
         <string>网络延时：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblVideoBitrate">
        <property name="text">
         <string>视频发送/接收码率：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblAudioBitrate">
        <property name="text">
         <string>音频发送/接收码率：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblVideoLossRate">
        <property name="text">
         <string>本地上行/下行视频丢包率：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblAudioLossRate">
        <property name="text">
         <string>本地上行/下行音频丢包率：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblCpu">
        <property name="text">
         <string>CPU占用 总/app：</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
